app/src/main/java/org/greenstand/android/TreeTracker/splash/SplashScreenPreviewProvider.kt

```diff
@@ -0,0 +1,61 @@
+package org.greenstand.android.TreeTracker.splash
+
+import android.content.SharedPreferences
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import org.greenstand.android.TreeTracker.preferences.Preferences
+import org.greenstand.android.TreeTracker.preferences.PreferencesMigrator
+
+class SplashScreenPreviewProvider : PreviewParameterProvider<SplashScreenViewModel> {
+
+    private val stubSharedPrefs = object : SharedPreferences {
```

Comment:
We should make a single stub version of this that we can reuse.

Reply:
I was thinking this as well. I noticed I was repeating code and will likely need to repeat this again. I think I'll make a `PreviewProviderUtil` class or something with common objects that be used as stubs